content/02.delete-me.md

```diff
@@ -217,6 +226,15 @@ _Journals are not likely to support them, and they may not display correctly whe
 
 [Link styled as a button](https://manubot.org "Manubot Homepage"){.button}
 
+Adding arbitrary HTML attributes with the `attributes` plugin:
+
+Manubot Manubot Manubot Manubot Manubot.
+Manubot Manubot Manubot Manubot.
+Manubot Manubot Manubot.
+Manubot Manubot.
+Manubot.
+<!-- $id="paragraph_of_text" $style="color: #ad1457; margin-left: 40px;" $disabled="true" $title="a paragraph of text" $data-color="red" -->
```

Comment:
In this case, pandoc syntax could also be used to set these values, so should we add a note that says something like "for demonstration, since these attributions can also be set using pandoc extensions or html syntax"?

Reply:
I’d rather also have a paragrah right above this one demonstrating the pandoc syntax

Reply:
I’d rather also have a paragrah right above this one demonstrating the pandoc syntax